### PR TITLE
feat(slugrunner): pass storage parameters to slugrunner

### LINKED ISSUE #458: change secrets to access object storage credentials
In build pack apps template and in creating a secret a object store secret in the App namespace we are still using minio-user . Have to use deis-object-storage secret

### LINKED ISSUE #521: Swap "minio" for generic "object storage"
See here: https://github.com/deis/controller/blob/745f7e1504ae6cb9374a94d77f37eabe48578788/rootfs/scheduler/__init__.py#L46-L66

It would probably be more proper to not use "minio" here and, instead, not assume or hint at the use of any particular object storage provider by using a more generic term like "object storage" instead.

A large benefit to this is cleaner charts.

cc @arschles @helgi

### PR DESCRIPTION
**Pull request by kmala**
pass storage parameters to slugrunner

Fixes #458
Fixes #521

### CODE REVIEW

**Comment by arschles** [comment]
This requires https://github.com/deis/slugrunner/pull/24, right?

**Comment by kmala** [comment]
Yes. this requires the slugrunner PR

**Comment by smothiki** [comment]
Also dont merge this untill slugrunner PR gets merged

File: rootfs/scheduler/__init__.py [hunk 1]
```diff
@@ -41,14 +41,14 @@
             "value":"$image"
         },
         {
-            "name": "DOCKERIMAGE",
```
Comment (helgi) [inline rootfs/scheduler/__init__.py]: This information is no longer required?

Comment (smothiki) [inline rootfs/scheduler/__init__.py]: Yes we have moved away from MC binary

**Comment by helgi** [comment]
Does this PR closes https://github.com/deis/controller/issues/458 and https://github.com/deis/controller/issues/521

File: rootfs/scheduler/__init__.py [hunk 2]
```diff
@@ -194,14 +194,22 @@
                 "value":"$appversion"
             },
             {
-                "name": "DOCKERIMAGE",
-                "value":"1"
+                "name": "BUILDER_STORAGE",
+                "value":"$storagetype"
+            },
+            {
+                "name": "DEIS_MINIO_SERVICE_HOST",
```
Comment (helgi) [inline rootfs/scheduler/__init__.py]: Are these two ENV vars not auto exposed to the service? Does that just happen within a given namespace?

**Comment by smothiki** [comment]
>Does this PR closes #458 and #521

yes

File: rootfs/scheduler/__init__.py (hunk 2 above)
Comment (kmala) [inline rootfs/scheduler/__init__.py]: yes

Comment (helgi) [inline rootfs/scheduler/__init__.py]: Fair enough, not super thrilled having more information in there but it's required so YOLO!

**Pull request merged**